Set up Hacktoberfest 2023 infrastructure (#4818)
withastro/docs: .github/hacktoberfest.md

3. Let us know you’re taking part by requesting for your PR to be labeled as `hacktoberfest-accepted`.

Remember, we need to accept your PR by 31 October 2023 for it to count towards Hacktoberfest. Have your PR ready as early as possible, so that you can address comments and have it approved before the deadline.

## How can I contribute?

Hacktoberfest [encourages translations and other low or no-code contributions](https://hacktoberfest.com/participation/#low-or-non-code), and for that reason, our event is primarily focused on internationalization contributions. To help you, the Astro Team has chosen a few "themes" to inspire your contributions this month! We also highly recommend [joining us on our Discord community](https://astro.build/chat/), there we can help you get started and avoid having any duplicate efforts. 

### Themes

#### Stars Around the Globe

Astro's official docs framework Starlight welcomes UI translations for any language! 

Learn more on how to contribute with a UI translation in [Starlight's contribution guide](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md#translating-starlights-ui).

#### World Tour Comeback 

> A few languages of Astro Docs' are behind its colleagues - What are you waiting for?! Help Arabic, Korean, Traditional Chinese, and Polski docs come back into the fight!

Reviewer: ```suggestion
A few languages of Astro Docs' are behind its colleagues - What are you waiting for?! Help Arabic, Korean, Deustch, French, Russian, and Polski docs come back into the fight!
```

> A few languages of Astro Docs' are behind its colleagues - What are you waiting for?! Help Arabic, Korean, Traditional Chinese, and Polski docs come back into the fight!

Reviewer 2: A couple of grammar nits and what do you think of this slightly less adversarial framing?

```suggestion
Some languages in Astro’s Docs need a boost. What are you waiting for?! Help Arabic, Korean, Traditional Chinese, and Polski docs come back into the fight!
```

I’ll let you decide if the other languages @dreyfus92 suggested need adding or not. We should keep Traditional Chinese though.

> A few languages of Astro Docs' are behind its colleagues - What are you waiting for?! Help Arabic, Korean, Deustch, French, Russian, and Polski docs come back into the fight!

Author: ```suggestion
Some languages in Astro’s Docs need a boost. What are you waiting for?! Help Arabic, Korean, Traditional Chinese, Deutsch, French, Russian, and Polski docs come back into the fight!
```

> Learn more on [Astro Docs' translation guide](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md).

Reviewer 2: Maybe I’m wrong, but in general I think of it as “Astro’s Docs” or the “Astro docs repo” rather than “Astro Docs”. @sarah11918 is probably best placed to confirm.

In this case we can probably avoid it anyway:

```suggestion
Learn more in our [translation guide](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md).
```

#### Polyglot Astro 101 

Astro Docs features its own 32-part beginner tutorial in several languages. Let's help other newcomers learn Astro around the world today! 

> Learn more on [Astro Docs' translation guide](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md) and in [its own Tutorials section](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md#tutorials).

Reviewer 2: Love this framing, but are we sure the tutorial is a sensible thing to call out for Hacktoberfest? It’s often a long road to complete the translation and could easily overrun the deadline for contributions, which might be a frustrating experience if someone was hoping to have the contribution count.

Author: Fair, also considering PRs to tutorial branches probably aren't counted for the label, let's remove it then!
```suggestion

```

> #### Here Comes a New Fighter!

Author: Maybe I went too deep into the Street Fighter reference here 😝

Astro Docs just added official support for Hindi translations. Let's show how strong its community is!

Learn more on [Astro Docs' translation guide](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md).

#### Global Level Reviews 

No translation is approved without a careful double-check! Review a page and increase the document's [i18n reviews count](#i18n-reviews-count).

> Learn more on [Astro Docs' translation guidance for reviewers](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md#reviewing-someone-elses-pr).

Reviewer 2: ```suggestion
Learn more in our [translation guidance for reviewers](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md#reviewing-someone-elses-pr).
```

## i18n Reviews Count

Reviewed a PR? Create a PR increasing the count below to have your contribution considered!

> During Hacktoberfest 2023, we've received 0 PR reviews from helpful contributors. Thank you!

Reviewer 2: I know there was some discussion around whether the list format we used last year was hard to manage. FWIW this format _guarantees_ a merge conflict between updates. That’s OK if we prefer handling that manually each time and like this format, but flagging it just so we’re all aware.

Reviewer 3: I'm pretty sure the list format, in practice, guaranteed a merge conflict every time.  :sweat_smile: 

We have to stop and confirm the contribution by looking up the PR anyway, so updating a number doesn't feel nearly as much work as the list from last time. (Still remember the same person who added themselves to "the end of the same list" in the same spot 3 times for 3 PRs... Still open for suggestions though, especially for whatever data we want to capture!

e.g. Did we want to do these per-language, too? Have a count for each language?

Author: > e.g. Did we want to do these per-language, too? Have a count for each language?

I feel like just having the count is enough.
